public/js/monitor.js

```diff
@@ -213,6 +213,14 @@ function styleActiveLink(locationHref) {
 ( async() => {
   document.addEventListener("touchstart", function(){}, true);
   const win = window;
+  if (win.navigator.language.startsWith("de")) {
+   const headlineElsToRestyle = document.querySelectorAll("[data-lang]");
```

Comment:
I did a quick <kbd>git grep "data-lang"</kbd> in **master** branch and didn't see any results, so I _think_ this is the only place we're currently using it. But... Should this be more specific?
`document.querySelectorAll(".feature-tip-headline[data-lang]")` to match the CSS selector above?